src/Resources/contao/dca/tl_faq_category.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * SMARTGEAR for Contao Open Source CMS
+ * Copyright (c) 2015-2022 Web ex Machina
+ *
+ * @category ContaoBundle
+ * @package  Web-Ex-Machina/contao-smartgear
+ * @author   Web ex Machina <contact@webexmachina.fr>
+ * @link     https://github.com/Web-Ex-Machina/contao-smartgear/
+ */
+
+use Contao\CoreBundle\Exception\AccessDeniedException;
+use Contao\Image;
+use Contao\Input;
+use Contao\System;
+use WEM\SmartgearBundle\Classes\Dca\Manipulator as DCAManipulator;
+
+DCAManipulator::create('tl_faq_category')
```

Comment:
Might be better to use dedicated classes instead of putting classes into these files
Ex: `WEM\SmartgearBundle\Backend\Datacontainer\Faq\Category::class, 'checkPermission'`

Reply:
Good idea ! Created an issue for that : https://github.com/Web-Ex-Machina/contao-smartgear/issues/91